samples/nextjs/src/pages/[[...path]].tsx

```diff
@@ -36,14 +39,28 @@ const SitecorePage = ({ notFound, layoutData, componentProps }: SitecorePageProp
 
 // This function gets called at build and export time to determine
 // pages for SSG ("paths", as tokenized array).
-export const getStaticPaths: GetStaticPaths = async () => {
+export const getStaticPaths: GetStaticPaths = async (context) => {
   // Fallback, along with revalidate in getStaticProps (below),
   // enables Incremental Static Regeneration. This allows us to
   // leave certain (or all) paths empty if desired and static pages
   // will be generated on request.
   // See https://nextjs.org/docs/basic-features/data-fetching#incremental-static-regeneration
   //
-  // Ultimately, this is where we'll also be able to request a "sitemap" from Sitecore.
+
+  if (process.env.BUILD_MODE === 'export') {
```

Comment:
As discussed, we don't have to worry about export in our sample, but we should definitely have a condition here to enable/disable the sitemap fetch, especially in development since it [runs on every request](https://nextjs.org/docs/basic-features/data-fetching#runs-on-every-request-in-development-1). 

Could keep it simple and just use `NODE_ENV` for now:
 ```
  if (process.env.NODE_ENV !== 'development') {
```
Could also introduce a new .env variable, but would need to have comments there to document usage (not needed for SSR, etc).

Reply:
The locale is still needed/desired for Incremental SSG right?

Reply:
@ambrauer I will use `NODE_ENV` it doesn't make sense to define separate variable. If somebody will need it - they can define it.

@nickwesselman Yes, I'm using `locale` property.